app/authenticators/osf-cookie.ts

```diff
@@ -86,7 +86,7 @@ export default class OsfCookie extends Base {
         return { id };
     }
 
-    restore() {
+    async restore() {
```

Comment:
Mostly because the ember-simple-auth upgrade guide said to do so.  https://github.com/mainmatter/ember-simple-auth/blob/master/guides/upgrade-to-v3.md#make-your-custom-session-store-asynchronous

Reply:
In which case, should we `return await this.authenticate()`

Reply:
Yeah, I can see why it didn't complain, because it *is* returning a promise, but this is the more correct way to go.